src/Toolkit/Toolkit/LocalizedStrings/Resources.da.resx

```diff
@@ -118,267 +118,267 @@
   </data>
   <data name="FeatureFormIncorrectValueTypeFloatingPoint" xml:space="preserve">
-    <value>ø_Value must be a number_______________________å</value>
+    <value>Værdien skal være et tal</value>
   </data>
   <data name="FeatureFormIncorrectValueTypeNumeric" xml:space="preserve">
-    <value>ø_Value must be a whole number_____________________________å</value>
+    <value>Værdien skal være et helt tal</value>
   </data>
   <data name="FeatureFormLessThanMinimumDateTime" xml:space="preserve">
-    <value>ø_Date must be on or after {0}_____________________________å</value>
+    <value>Dato skal være på eller efter {0}</value>
   </data>
   <data name="FeatureFormLessThanMinimumDateTimeNoRange" xml:space="preserve">
-    <value>ø_Date less than minimum_______________________å</value>
+    <value>Dato mindre end minimumværdien</value>
   </data>
   <data name="FeatureFormLessThanMinimumLength" xml:space="preserve">
-    <value>ø_Minimum characters required: {0}_________________å</value>
+    <value>Minimum antal påkrævede tegn: {0}</value>
   </data>
   <data name="FeatureFormLessThanNumericMinimum" xml:space="preserve">
-    <value>ø_Less than minimum value________________________å</value>
+    <value>Mindre en minimumværdien</value>
   </data>
   <data name="FeatureFormNotInCodedValueDomain" xml:space="preserve">
-    <value>ø_Value must be within the domain_________________å</value>
+    <value>Værdi skal være inden for domænet</value>
   </data>
   <data name="FeatureFormNullNotAllowed" xml:space="preserve">
-    <value>ø_Value must not be empty________________________å</value>
+    <value>Værdi må ikke være tom</value>
   </data>
   <data name="FeatureFormOutsideLengthRange" xml:space="preserve">
-    <value>ø_Enter {0} to {1} characters____________________________å</value>
+    <value>Indtast fra {0} til {1} tegn</value>
   </data>
   <data name="FeatureFormSpecificLength" xml:space="preserve">
-    <value>ø_Number of characters required: {0}__________________å</value>
+    <value>Antal påkrævede tegn: {0}</value>
   </data>
   <data name="FloorFilterAllFacilities" xml:space="preserve">
-    <value>ø_All Sites___________________å</value>
+    <value>Alle sites</value>
   </data>
   <data name="FloorFilterAllFloors" xml:space="preserve">
-    <value>ø_All Floors_____________________å</value>
+    <value>Alle etager</value>
   </data>
   <data name="FloorFilterBackButton" xml:space="preserve">
-    <value>ø_Go Back_______________å</value>
+    <value>Gå tilbage</value>
   </data>
   <data name="FloorFilterBrowse" xml:space="preserve">
-    <value>ø_Browse_____________å</value>
+    <value>Gennemse</value>
   </data>
   <data name="FloorFilterBrowseFacilities" xml:space="preserve">
-    <value>ø_Select a facility__________________å</value>
+    <value>Vælg en facilitet</value>
   </data>
   <data name="FloorFilterBrowseSites" xml:space="preserve">
-    <value>ø_Select a site______________å</value>
+    <value>Vælg et websted</value>
```

Comment:
This was translated to "website" which is incorrect. I wonder if other languages might have the same issue.